runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -58,7 +58,12 @@ async def explain_v1_output(self, request: InferenceRequest) -> Response:
         in the endpoint.
         """
         v2_response = await self.predict(request)
-        explanation = json.loads(v2_response.outputs[0].data.__root__)
+        output = v2_response.outputs[0]
+
+        # TODO: Does this default value make sense here?
+        explanation = {}
```

Comment:
perhaps we should raise an exception if we cannot recover the explanation dict?

Reply:
Good point! Let me do that :+1: